External/FEXCore/Source/Interface/Core/Dispatcher/Dispatcher.cpp

```diff
@@ -274,6 +289,19 @@ static uint32_t ConvertSignalToError(int Signal, siginfo_t *HostSigInfo) {
   return 0;
 }
 
+template <typename T>
+static void SetXStateInfo(T* xstate) {
```

Comment:
This function doesn't seem to be used.

Reply:
Ah good catch, I forgot to add the usages back after finishing testing